src/styles/index.scss

```
@@ -1,3 +1,153 @@
+$font-family-primary: 'Roboto', Arial, sans-serif;
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: 400px;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
```

```suggestion
  &__image {
```
it's an element of `product-card` block, so there is no need to duplicate `product` in name